docs/content/en/docs/concepts/eksapackages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "EKS Anywhere curated packages"
+linkTitle: "EKS Anywhere curated packages"
+weight: 4
+date: 2022-04-12
+description: >
+  Explanation of the process of managing curated packages in an EKS Anywhere cluster
+---
+
+EKS Anywhere curated packages is a framework to manage installation, configuration and maintenance of components that provide general operational capabilities for Kubernetes applications. Examples of these components include CSI, Ingress, LoadBalancer, Container Registry, Service Mesh, Secret Management, etc.
+
+## Overview
+The major components of EKS Anywhere curated packages are the package controller, the package build artifacts and the command line interface. The package controller will run in a pod in an EKS Anywhere cluster. The package controller will install packages from EKS Anywhere public ECR repository of tested package images.
```

Comment:
`The package controller will manage the lifecycle of packages tested and maintained by EKS Anywhere`